src/blk/kernel/KernelDevice.cc

```diff
@@ -67,10 +67,12 @@ KernelDevice::KernelDevice(CephContext* cct, aio_callback_t cb, void *cbpriv, ai
   fd_buffereds.resize(WRITE_LIFE_MAX, -1);
 
   bool use_ioring = cct->_conf.get_val<bool>("bdev_ioring");
+  bool use_ioring_hipri = cct->_conf.get_val<bool>("bdev_ioring_hipri");
+  bool use_ioring_sqthread_poll = cct->_conf.get_val<bool>("bdev_ioring_sqthread_poll");
```

Comment:
could you move these two lines into the `if (use_ioring && ioring_queue_t::supported()) {` block?

Reply:
Ok, no problem.